genomesubsampler/filterRepophlan.py

```diff
@@ -0,0 +1,101 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017--, genome-subsampler development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+"""
+Program to filter repophlan results by the quality scores.
+
+Example usage:
+
+    filter_repophlan.py -r ./repophlan_microbes_wscores.txt -s 0.6
+
+Will produce two files:
+
+    repophlan_filepaths.good and repophlan_filepaths.bad
+
+each will contain the filenames in the following columns:
+
+    faa_lname,ffn_lname,fna_lname,frn_lname
+
+as well as the original score values and the summarized score.
+
+Score can be calculated as in the repophlan paper, using a normalized and
+averaged score, or by passing the -a/--avg flag, to just use an average of
+each of the four scores per genome.
+"""
+
+import pandas as pd
+import click
+
+
+def calc_norm_score(genome_df, cols=['score_faa',
+                                     'score_fna', 'score_rrna', 'score_trna']):
+    sub_df = genome_df[cols].copy()
```

Comment:
can we please have a doc string for very function?
What are the parameters, what is the return value and what does the function do?